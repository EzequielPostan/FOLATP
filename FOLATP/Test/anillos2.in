loadQuery( Test/anillos.kb )
conclusion( Odia(Frodo,Sauron) )
runTo( Test/anillos2.out )
exit
