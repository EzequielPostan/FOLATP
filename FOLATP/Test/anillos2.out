{}
│
├─ {ViveEn(Frodo, TierraMedia)}
│ │
│ ├─ {ViveEn(x1, TierraMedia), ~Hobbit(x1)}
│ │ │
│ │ ├─ {ViveEn(x1, Comarca), ~Hobbit(x1)}
│ │ │
│ │ └─ {ViveEn(x1, TierraMedia), ~ViveEn(x1, Comarca)}
│ │
│ └─ {Hobbit(Frodo)}
│
└─ {~ViveEn(Frodo, TierraMedia)}
  │
  ├─ {EsLeal(Frodo, Sauron), ~ViveEn(Frodo, TierraMedia)}
  │ │
  │ ├─ {EsLeal(x1, Sauron), Odia(x1, Sauron), ~ViveEn(x1, TierraMedia)}
  │ │
  │ └─ {~Odia(Frodo, Sauron)}
  │
  └─ {~EsLeal(Frodo, Sauron)}
    │
    ├─ {IntentaDestruir(Frodo, Sauron)}
    │
    └─ {~EsLeal(x1, x2), ~IntentaDestruir(x1, x2)}
